src/test/java/com/stripe/functional/CardTest.java

```diff
@@ -1,20 +1,150 @@
 package com.stripe.functional;
 
-import com.stripe.BaseStripeFunctionalTest;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import com.stripe.BaseStripeTest;
 import com.stripe.exception.StripeException;
+import com.stripe.model.Card;
 import com.stripe.model.Customer;
+import com.stripe.model.DeletedCard;
+import com.stripe.model.ExternalAccount;
+import com.stripe.model.ExternalAccountCollection;
+import com.stripe.net.APIResource;
 
+import java.io.IOException;
 import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
 import java.util.Map;
 
 import org.junit.Test;
 
-public class CardTest extends BaseStripeFunctionalTest {
+public class CardTest extends BaseStripeTest {
+  public static final String CUSTOMER_ID = "cus_123";
+  public static final String CARD_ID = "card_123";
+
+  private Card getCardFixture(Customer customer) throws IOException, StripeException {
+    final Card card = APIResource.GSON.fromJson(
+        getResourceAsString("/api_fixtures/card.json"), Card.class);
+    card.setCustomer(customer.getId());
+
+    return card;
```

Comment:
you likely want a comment to explain that this is because stripe-mock is mean to you.